mutiny-core/src/lib.rs

```diff
@@ -1062,6 +1062,24 @@ impl<S: MutinyStorage> MutinyWallet<S> {
                                                         log_error!(logger, "Error handling dm: {e}");
                                                 }
                                             }
+                                            Kind::ContactList => {
+                                                match update_nostr_contact_list(&nostr.storage, *event) {
+                                                    Err(e) =>log_error!(logger, "Error handling contact list: {e}"),
+                                                    Ok(true) => {
+                                                        log_debug!(logger, "Got new contact list, syncing...");
+
+                                                        // sync in background so we don't block processing other events
+                                                        let self_clone = self_clone.clone();
+                                                        utils::spawn(async move {
+                                                            match self_clone.sync_nostr_contacts(self_clone.nostr.public_key).await {
+                                                                Err(e) => log_error!(self_clone.logger, "Failed to sync nostr: {e}"),
+                                                                Ok(_) => log_debug!(self_clone.logger, "Successfully synced nostr contacts"),
+                                                            }
+                                                        });
+                                                    }
+                                                    Ok(false) => log_debug!(logger, "Got older contact list, ignoring..."),
```

Comment:
Is there no other quicker way to have the update reflected in the contacts list than doing a full resync?

Reply:
I guess we could diff our current list and sync the individual profiles.

Reply:
What does the sync actually do?

Reply:
Pulls down the their profile and sets in our storage. So we have their name, pgp, lightning addr, etc.

Reply:
Gotcha. 

Couldn't we just do that for once for one profile? Instead of the whole list or trying to do a diff? It should just be adding/removing one at a time.

Reply:
Yeah, this listener is for if a change is made outside the app.

Reply:
Oh, damn, didn't expand the top. Nevermind, that is fine I think.